roles/clevis/vars/Debian.yml

```diff
@@ -0,0 +1,10 @@
+---
+clevis_packages:
+  - clevis
+  - clevis-dracut
+  - clevis-initramfs
+  - clevis-luks
```

Comment:
This looks the same as for redhat, does this really need to be distro specific? Being able to use a generic default would be much easier.